### PR TITLE
adding logic to show, edit and remove  (existing) contacts

### PR DESCRIPTION
**Pull request by brianstanley**
issue #26

### CODE REVIEW

File: src/components/layout/AddContacts.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<script>
```
Comment (ianaya89) [inline src/components/layout/AddContacts.vue]: I think that `Contacts.vue` will be more proper for the component name.

Comment (ianaya89) [inline src/components/layout/AddContacts.vue]: @matiastucci @ndelvalle what do you thin?

Comment (ndelvalle) [inline src/components/layout/AddContacts.vue]: Yep, I agree 👍

Comment (brianstanley) [inline src/components/layout/AddContacts.vue]: 👍

**Pull request merged**